docker-context/Dockerfile

```diff
@@ -30,25 +30,37 @@ RUN { \
 		echo 'memory_limit = 1024M'; \
 	} > /usr/local/etc/php/conf.d/tap.ini;
 RUN wp --allow-root package install https://github.com/nlemoine/wp-cli-fixtures/archive/refs/tags/0.3.0.zip
+COPY apache_port_template.conf /etc/apache2/ports.conf
+# Make sure the port override below matches the docker-compose port for wp49-php72
+RUN sed -i 's/<PORT>/8031/g' /etc/apache2/ports.conf
```

Comment:
To avoid having a separate URL for each WP service, we tell each WP service to listen to the ports we define in our docker-compose file.